traits/src/router.rs

```diff
@@ -68,6 +105,31 @@ pub enum PoolType<AssetId> {
 	Omnipool,
 }
 
+#[derive(Encode, Decode, Clone, Copy, Debug, Eq, PartialEq, TypeInfo, MaxEncodedLen)]
+pub enum AssetType<AssetId, NFTId> {
+	Fungible(AssetId),
+	NFT(NFTId),
```

Comment:
why not call it explicitly - Nonfungible ?

Reply:
since we removed, it is not applicable anymore